docs/source/devguide/index.rst

```diff
@@ -110,3 +110,60 @@ Implementing a new feature
 #. :ref:`Make your changes<contributing to the code>`
 
 #. Write a test to test your feature
+
+
+-------------------
+Documentation guide
+-------------------
+
+The documentation is a crucial part of the project. It is the first thing users will see when they want to use FESTIM.
+It is important to keep it up to date and clear.
+
+The documentation is written in `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_ and is located in the `docs folder <https://github.com/festim-dev/FESTIM/tree/main/docs>`_ at the root of the FESTIM repository.
+
+The documentation is built using `Sphinx <https://www.sphinx-doc.org/en/master/>`_.
+
+To build the documentation locally, you can use the following command:
+
+.. code-block:: bash
+
+   cd docs/source
+   make html
+
+This will generate the documentation in the `docs/source/_build/html` folder.
+You can then open the `index.html` file in your browser to see the documentation.
```

Comment:
Maybe, `make clean` is also worth mentioning?

Reply:
Hm maybe, what does it do? I usually use the `sphinx-build` command

Reply:
The list of available arguments for `make`:
```
Please use `make target' where target is one of
  html        to make standalone HTML files
  dirhtml     to make HTML files named index.html in directories
  singlehtml  to make a single large HTML file
  pickle      to make pickle files
  json        to make JSON files
  htmlhelp    to make HTML files and an HTML help project
  qthelp      to make HTML files and a qthelp project
  devhelp     to make HTML files and a Devhelp project
  epub        to make an epub
  latex       to make LaTeX files, you can set PAPER=a4 or PAPER=letter
  latexpdf    to make LaTeX and PDF files (default pdflatex)
  latexpdfja  to make LaTeX files and run them through platex/dvipdfmx
  text        to make text files
  man         to make manual pages
  texinfo     to make Texinfo files
  info        to make Texinfo files and run them through makeinfo
  gettext     to make PO message catalogs
  changes     to make an overview of all changed/added/deprecated items
  xml         to make Docutils-native XML files
  pseudoxml   to make pseudoxml-XML files for display purposes
  linkcheck   to check all external links for integrity
  doctest     to run all doctests embedded in the documentation (if enabled)
  coverage    to run coverage check of the documentation (if enabled)
  clean       to remove everything in the build directory
```

Reply:
Oh yes I remembered I used `make doctest` for another project. Probably need to use this for #718